baracoda/sql/schema.sql

```diff
@@ -10,6 +10,10 @@ DROP SEQUENCE IF EXISTS sqp;
 CREATE SEQUENCE sqp
 START 1;
 
+DROP SEQUENCE IF EXISTS csm;
+CREATE SEQUENCE csm
+    START 111111;
+
```

Comment:
This is the change from one of the new migrations, but the one adding the index to the table is not shown here.